tests/test_save_output.py

```diff
@@ -86,9 +86,14 @@ def test_save_output():
     print("Output loaded from h5 file")
 
     # compare scenes 
-    saved = jax.tree_util.tree_structure(scene_)
-    loaded = jax.tree_util.tree_structure(scene_loaded)
-    print(f"saved == loaded: {saved == loaded}")
+    saved = jax.tree_util.tree_leaves(scene_)
+    loaded = jax.tree_util.tree_leaves(scene_loaded)
+    status = True
+    for leaf_saved, leaf_loaded in zip(saved, loaded):
+        if (leaf_saved != leaf_loaded).all():
+            status = False
+        
+    print(f"saved == loaded: {status}")
```

Comment:
One last thing: to work as a test, it needs to be pass clean or raise an error, so use `assert status is True, "Loaded leaves not identical to original"` instead of  / in addition to  `print`